### DISCUSSION TITLE
Docker Compose with --always-cpu

### DISCUSSION THREAD

**Discussion by silvio-vallorani** [lllyasviel/Fooocus#3612] [Q&A] [answered]
How to modify the docker-compose.yml to get the Fooocus running in Docker in a PC without NVIDIA/AMD graphic card?
I read that without docker can be used the --always-cpu flag, is there a way to pass it to docker image using the Docker Compose?

Thank you.

**Comment by salman2learn** [comment, marked as answer]
I was searching for the same.
The way to do it is to modify docker-compose as below: (use your appropriate number instead of 8 for CPU threads)
```
- CMDARGS=--listen --always-cpu 8
```

**Reply by kevykibbz** [reply]
thanks.
